### PR TITLE
Update definitions.yaml

### PR DESCRIPTION
**Pull request by yoavrotems**
Update to use bitmask flag
https://github.com/aquasecurity/linux-bench/issues/52

### CODE REVIEW

File: cfg/2.0.0/definitions.yaml [hunk 1]
```diff
@@ -7033,13 +7033,16 @@ groups:
       description: "Ensure rsyslog default file permissions configured"
       sub_checks:
       - check:
-          audit: "grep ^\\$FileCreateMode /etc/rsyslog.conf /etc/rsyslog.d/*.conf"   
+          audit: "grep ^\\$FileCreateMode /etc/rsyslog.conf /etc/rsyslog.d/*.conf -h | tr \"$\" \" \" "
```
Comment (yoavrotems) [inline cfg/2.0.0/definitions.yaml:7036]: Removed file name and $ from the start of the output

Comment (simar7) [reply cfg/2.0.0/definitions.yaml:7036]: I would like to try this. I have a vagrant VM with a `/etc/rsyslog.conf` available. And this is what I see:

```
vagrant@vagrant:~$ grep ^\\$FileCreateMode /etc/rsyslog.conf -h | tr \"$\" \" \"
tr: extra operand ‘"’
Try 'tr --help' for more information.
grep: Trailing backslash
vagrant@vagrant:~$ grep ^\\$FileCreateMode /etc/rsyslog.conf -h | tr \"$\" \"
grep: Trailing backslash
vagrant@vagrant:~$ grep ^\\$FileCreateMode /etc/rsyslog.conf -h
grep: Trailing backslash
```

What am I doing wrong?

Comment (yoavrotems) [reply cfg/2.0.0/definitions.yaml:7036]: There is escaping, remove the back slash before special characters:
`grep ^\$FileCreateMode /etc/rsyslog.conf /etc/rsyslog.d/*.conf -h | tr "$" " "`

Comment (simar7) [reply cfg/2.0.0/definitions.yaml:7036]: OK, I'll have to take your word for it.

**Pull request merged**